index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+<h1>Pre-class Assignment</h1>
+<h2>about me </h2>
+	<p>I am Aziz a student at HYF. Working toward being a web developer. I passionate about problem solving and I also like creating project from scratch. Although I prefer working on functionality side of projects but it also gives me feelings of success and satisfaction when I create a whole project on my own. I like to have new challenges everyday ! That’s why I love coding! </p>
+	<h2>My goal</h2>
+`	<p>My short term goal is to be <em>a web developer</em>, but I also like to go further and have experience more depth in creating softwares. I would be working very hard to attain my goal and have a job that I enjoy everyday. I am striving to have a job at which I can do different tasks everyday and not just accomplishing simple tasks </p>
+
+<ul>
+	<li>Coffee</li>
+	<li>Politics</li>
+	<li>Fast food</li>
+</ul>
+
+<img src="images/Coffee.jpg" width="400" height="300">
+<br>
+<img src="images/huskey.jpeg" width="400" height="400">
+<br>
+<img src="images/ostrich.jpg" width="400">
+
+<br>
+Check my projcts: 
+<a href="https://www.github.com/azizck/"></a>
```

Comment:
Your link to github doesn't show up on your webpage. Do you know what's wrong, and how to fix it?